onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3443,15 +2889,10 @@ It's an extension of Gelu. It takes the sum of input A and bias input B as the i
   }
 #endif
 
-  RegisterNhwcSchemas();
```

Comment:
Does removing this provide any advantage? I thought it gave a nice way to see what contrib schemas are registered.

Reply:
Adding comments to [ms_opset.h](https://github.com/microsoft/onnxruntime/pull/10494/files/29a54edcf3adf2edb23b2f185d1bdf2ce99cc25d#diff-f1229ecff4afa1401adc6f3fc1f8db9909fe7ee6d9eda3bd0d8ac995b699051f) has the same effect. If you want to keep the structure, I can split ms_opset.h to smaller pieces.